### DISCUSSION TITLE
DOI and citations

### DISCUSSION THREAD

**Discussion by eileenrmartin** [DASDAE/dascore#79] [General]
Zack Spica contacted me about adding a proper citation to a note in the PubDAS paper that points people to the DASDAE project. I think it makes more sense to cite DASCore at this point than to cite DASDAE since it's a GitHub organization.

I've adjusted the DASDAE organization so it can interface with third party apps. This allowed me to make a zenodo DOI for a new release of DASCore, and then I added a link into the main readme.md file with the updated DOI and icon. 

It probably makes sense to cite any DASDAE packages so they're named Distributed Acoustic Sensing Data Analysis Ecosystem: PackageName. What do you all think? 

Also, I'd like to propose that we manage the list of co-authors by having all contributors listed from most-to-least lines added. It's maybe not a perfect system for capturing all contributions, but is objective, easy for any of us to check, and allows all of us an easy, transparent path to getting credit for your work.

Here's the current citation if we go with these policies: 

D. Chambers, G. Jin, A.H. Issah, E.R. Martin. 2022, Distributed Acoustic Sensing Data Analysis Ecosystem: DASCore, v0.0.7, Zenodo, doi: 10.5281/zenodo.7373559

Does anyone object to this, or have other policies to suggest for the citation?

**Comment by d-chambers** [comment]
Update on this. I verified that  Zenodo is archiving every new DASCore version. DASCore's page is found [here](https://zenodo.org/search?q=dascore&l=list&p=1&s=10&sort=bestmatch). 

In #317 I also added the Zenodo badge, so the image in the readme will always show the DOI for the latest version.

**Comment by d-chambers** [comment]
I think with the publication of the Seismica paper (and #423) and the Zenodo work described above we are good to close this. Feel free to re-open if there is more to do here @eileenrmartin .

**Reply by eileenrmartin** [reply]
I think we're good on this. One point that could be discussed is whether the simple system of lines contributed to the repo is the best way to attribute order (but I'm not sure of a better system that is easy to implement).
